test/acceptance/registry-ui.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+const expect = require('chai').expect;
+const oc = require('../../src/index');
+const path = require('path');
+const request = require('minimal-request');
+
+describe('registry (ui interface)', () => {
+
+  let registry, result, error, headers;
+
+  const next = (done) => (e, r, d) => {
+    error = e;
+    result = r;
+    headers = d.response.headers;
+    done();
+  };
+
+  const conf = {
+    local: true,
+    path: path.resolve('test/fixtures/components'),
+    port: 3030,
+    baseUrl: 'http://localhost:3030/',
```

Comment:
I've recently encountered a bug where `/v2/` was being removed when browsing components. But if it's covered elsewhere then it doesn't matter